examples/keygen/keyimport.c

```diff
@@ -55,6 +56,10 @@ int TPM2_Keyimport_Example(void* userCtx, int argc, char *argv[])
     TPM_ALG_ID paramEncAlg = TPM_ALG_NULL;
     WOLFTPM2_SESSION tpmSession;
     const char* outputFile = "keyblob.bin";
+    byte derEncode = 0;
```

Comment:
Wrap all this new PEM logic with `#if !defined(NO_FILESYSTEM) && !defined(NO_WRITE_TEMP_FILES)`